### DISCUSSION TITLE
computation for distance zonal stats to AOI (polygon)

### DISCUSSION THREAD

**Discussion by ghost** [thinkingmachines/geowrangler#154] [Q&A] [answered]
[feedback from usability tester] Question: For distance zonal stats to AOI (polygon), computation is based on distance to centroid or distance to the border/ perimeter? If distance to centroid, there are instances where nearest feature should have been counted as POI within the polygon.

Maybe show NA or null instead of zero if POI/ feature is inside the polygon. >=0 may be interpreted as distance from the border.

**Comment by butchtm** [comment, marked as answer]
**[Answer]** The computation is based on  the _distance between the AOIs and data's geometries_ (i.e. the borders/perimeters for areas).  If the data point or area intersects the AOI, the distance computed is `0.`  

The `NA` value is reserved for those cases where there are no data points or areas within the distance `max_col` parameter . This behavior is consistent with the `sjoin_nearest` method in Geopandas as [discussed here](https://geopandas.org/en/stable/docs/reference/api/geopandas.sjoin_nearest.html).
